Firmware/code.py

```diff
@@ -5,72 +5,36 @@
 from adafruit_hid.keyboard import Keyboard
 from adafruit_hid.keycode import Keycode
 
-#Setting pin varables
-Screen_1_Btn_Pin = board.GP1
-Screen_2_Btn_Pin = board.GP2
-Screen_3_Btn_Pin = board.GP3
-Screen_4_Btn_Pin = board.GP4
-Censor_Btn_Pin = board.GP15
+class BtnControl:
+    def __init__(self, name, pin, key):
+        self.name=name
+        self.btn=digitalio.DigitalInOut(pin)
+        self.btn.direction=digitalio.Direction.INPUT
+        self.btn.pull=digitalio.Pull.DOWN
+        self.key=key
+        self.last_value=False

```

Comment:
Given this variable doesn't remain a bool, it seems like it would be better to use None here.